### PR TITLE
feat(SpokePool): Remove fillCompleted param from handleAcrossMessage callback in fillUSS function

### PR DESCRIPTION
**Pull request by nicholaspai**

### CODE REVIEW

**Review by pxrl — APPROVED** [review]

**Review by pxrl — APPROVED** [review]

File: contracts/SpokePool.sol [hunk 1]
```diff
@@ -25,6 +25,15 @@ interface AcrossMessageHandler {
         address relayer,
         bytes memory message
     ) external;
+
+    // New function interface to be used with USS functions since fillCompleted no longer has any
+    // meetings now that partial fills are impossible.
```
Comment (mrice32) [inline contracts/SpokePool.sol:30]: meetings? Was that the intended word?

File: contracts/SpokePool.sol [hunk 2]
```diff
@@ -25,6 +25,15 @@ interface AcrossMessageHandler {
...
+    function handleUSSAcrossMessage(
```
Comment (mrice32) [inline contracts/SpokePool.sol:31]: We should probably loop in Ryan to make sure we feel good about changing this interface on integrators. This means that the moment this change goes into effect, all integrators must have implemented the new function (even if its alongside the old function). If they haven't their relays will revert.

**Review by mrice32 — COMMENTED** [review]
Looks great! One comment

File: contracts/SpokePool.sol (hunk 1 above)
Comment (nicholaspai) [reply contracts/SpokePool.sol:30]: ah, "meaning"

**Comment by nicholaspai** [comment]
Not merging until we confirm that integrators are able to support new interface

**Pull request merged**